manifests/s/SMPlayer/SMPlayer/23.12.0/SMPlayer.SMPlayer.installer.yaml

```
@@ -0,0 +1,43 @@
+# Created with komac v2.0.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SMPlayer.SMPlayer
+PackageVersion: 23.12.0
+Platform:
+- Windows.Desktop
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+InstallerSwitches:
+- Silent: /S
+- SilentWithProgress: /S

```

```suggestion
InstallerSwitches:
  Silent: /S
  SilentWithProgress: /S
```
I'm not a YAML expert, but copying from my notes suggests that this shouldn't have the dashes ( - ).